### DISCUSSION TITLE
Task / Details Template

### DISCUSSION THREAD

**Discussion by nflatrea** [SecurityBrewery/catalyst#1103] [Ideas]
Hi !

To provide a complete playbook workflow, we might need to create Templates for recurrent tickets and alerts.
When used, all data stored like tasks, links, files, comment, title can be used and overwritten. Tickets could also be cloned from another one in a signle button for ease of use.

I'm tinkering to make it work, might need a little time.

**Comment by cugu** [comment]
Hi @nflatrea, you could use a reaction that triggers on a new ticket to set default values for it, similar to https://catalyst.security-brewery.com/docs/catalyst/engineer/reaction/examples/assign_ticket

Example:
```python
import sys
import json
import datetime
import os

from pocketbase import PocketBase

# Parse the ticket from the input
ticket = json.loads(sys.argv[1])

# Connect to the PocketBase server
client = PocketBase('http://127.0.0.1:8090')
client.auth_store.save(token=os.environ["CATALYST_TOKEN"])

# Update the ticket severity to "Low" if it is an alert
if ticket["record"]["type"] == "alert":
	client.collection("tickets").update(ticket["record"]["id"], {
		"state": {
			"severity": "Low",
		}
	})

# Create a timeline entry for the ticket
client.collection("timeline").create({
	"ticket": ticket["record"]["id"],
	"time": datetime.datetime.now().isoformat(),
    "message": "Ticket was created",
})
```

**Reply by nflatrea** [reply]
Yes figured out a workaround, Nevertheless would be nice to have something easy to use directly by default
